### PR TITLE
Upgrade protobuf to 1.6.

### PR DESCRIPTION
**Pull request by hicqu**
Upgrade protobuf to 1.6.

### CODE REVIEW

**Comment by BusyJay** [comment]
I don't think it's a good idea. This will require `protoc` on the build machine and also update source directly at compile time. The proto file barely changes, it doesn't have to be rebuild again and again.

**Comment by siddontang** [comment]
LGTM

**Comment by siddontang** [comment]
PTAL @BusyJay

**Comment by Hoverbear** [comment]
@hicqu This looks ok but I'd also like to see some instruction how to regenerate the proto if needed.

**Comment by siddontang** [comment]
any update?

**Comment by hicqu** [comment]
PTAL @Hoverbear .

File: README.md [hunk 1]
```diff
@@ -52,6 +52,14 @@ You may optionally want to install `cargo-check` to allow for automated rebuildi
 cargo watch -s "cargo check --features dev"
 ```
 
+If proto file `eraftpb.proto` changed, run the command to regenerate `eraftpb.rs`:
+
+```bash
+protoc proto/eraftpb.proto --rust_out=src
```
Comment (siddontang) [inline README.md]: no need to set the output path?

**Pull request merged**